pkg/controller/sdk/status_accessor_test.go

```diff
@@ -1,114 +0,0 @@
-/*
```

Comment:
Why did these tests get deleted but not the code under test? Is that code unused? Are the tests invalid?

Reply:
the entire sdk needs to get deleted and I did not want to migrate the tests off the v1alpha1 Status

Reply:
IMO if the code is still used, it should have tests.

Reply:
https://github.com/knative/eventing-contrib/issues/947